src/_locales/es/messages.json

```diff
@@ -268,7 +268,7 @@
         }
     },
     "badger_status_cookieblock": {
-        "message": "Cookies bloqueadas de $DOMAIN$",
+        "message": "Las cookies bloqueadas en $DOMAIN$",
```

Comment:
Does this mean, cookies are being blocked on example.com, or cookies of example.com are being blocked? We want to say that cookies of example.com are being blocked.

Reply:
The translation is OK but "Las" is not really necessary. I'll remove it.

Reply:
"Cookies bloqueadas en example.com" Means "Cookies blocked in example.com.
The better translation would be "Cookies de $DOMAIN$ bloqueadas" or "Cookies bloqueadas de $DOMAIN". First option sounds better to me.

Reply:
If I understand it correctly, I don't think so. "Cookies bloquedas en $DOMAINS" means "Las cookies están bloqueadas en $DOMAIN" but in a shorter form. You're stating that the cookies are blocked there. "Cookies de $DOMAIN bloqueadas" and "Cookies bloqueadas de $DOMAIN" would be right if you had a list of blocked cookies for the domain and that were the caption or title for the list. It's like you're saying "Hey, this are the cookies that Badger has blocked at $DOMAIN".

Reply:
For example say you visit example.com and Badger detects a third party, cdn1.com, that is in the yellow list.
When ever you visit a site that loads cdn1.com as a third party, cdn1.com will be cookie blocked.

"Cookies bloquedas en $DOMAINS" could be correct, but it's ambiguous and it could also be interpreted as if this would happen if it was a first-party. Adding "de" me makes it less ambiguous as "de" indicates possession (in this case by the third party domain) in that context as a third party and "en" indicates where or when what the verb is referring to happens (which would include when such third party is a first party).

Reply:
Ah, ok, I now get it. Thanks for your clarification. Fixed.